el8/system/init.d/kazoo-haproxy.redhat

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+#
+# haproxy
+#
+# chkconfig:   - 85 15
+# description:	HAProxy is a free, very fast and reliable solution \
+#				offering high availability, load balancing, and \
+#				proxying for TCP and  HTTP-based applications
+# processname: haproxy
+# config:	   /etc/kazoo/haproxy/haproxy.cfg
+# pidfile:	   /var/run/haproxy/kazoo-haproxy.pid
```

Comment:
This seems to be using the old path

Reply:
should probably just remove this, we dont use init.d in el8 anywa